### DISCUSSION TITLE
Schaufel Position

### DISCUSSION THREAD

**Discussion by Aichinto** [Courseplay/courseplay#7638] [Ideas]
Hallo zusammen, ich habe gesehen das ihr eure funktion " per Mausklick in ein anderes Fahrzeug switchen" aus coursplay, als seperaten Mod im modhub angeboten habt.

Meine Frage hierzu, besteh die Möglichkeit sowas auch mit den schaufel Positionen zu machen?
Habe im 19er oft euer menü dafür verwendet um vordefinierte Positionen einzustellen und damit zu arbeiten.

Vielen dank für eure tolle Leistung im Bereich moddign👍

**Comment by Tensuko** [comment]
Du meinst per Tasten die Schaufelpositionen anfahren ?
Das lässt sich bestimmt einrichten.
Ob es eine extra Mod geben wird, kann ich dir nicht sagen, da unser Focus jetzt auf CP für FS22 liegt.

**Reply by Aichinto** [reply]
Ja z.b, ja das wäre auch ne gute Lösung. 
Hab oft die unter Position  zum laden und auskippen genutzt um hänger zu beladen, wenn man das jetzt mittels Tastendruck ( z.b Tasten strg+ 1) das ausführen könnte ohne das cp menü zu öffnen

**Reply by Tensuko** [reply]
Es könnte sein, dass da demnächst was von uns kommt :)

**Reply by Aichinto** [reply]
Ouhha😏👍 ja sehr geil, ich bin gespannt 👌

**Comment by Tensuko** [comment]
Mod ist im ModHub eingereicht.

**Reply by Aichinto** [reply]
Ah sehr cool, auf github auch?

**Reply by Tensuko** [reply]
Nein, ModHub.
